debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/xstream/LcrEventHandler.java

```diff
@@ -133,7 +133,7 @@ private TableId getTableId(LCR lcr) {
             return new TableId(lcr.getSourceDatabaseName(), lcr.getObjectOwner(), lcr.getObjectName());
         }
         else {
-            return new TableId(lcr.getSourceDatabaseName().toLowerCase(), lcr.getObjectOwner(), lcr.getObjectName().toLowerCase());
+            return new TableId(lcr.getSourceDatabaseName(), lcr.getObjectOwner(), lcr.getObjectName().toLowerCase());
```

Comment:
Why is the table name still converted to lowercase?

Reply:
So the XStream adapter still returns `true` if the database is Oracle 11 and the `toLowerCase` on TableId only applies to the actual table-name.  The catalog and schema portions aren't changed.  The removal of the lowering of the catalog was to address a bug report with DBZ-2203 only.